src/app/core/theme.component.ts

```diff
@@ -60,6 +68,15 @@ import { ToggleComponent } from '@shared/ui/toggle/toggle.component';
     <mat-list class="list">
       <app-members-list-item [user]="user"></app-members-list-item>
     </mat-list>
+    <h2>Przyciski</h2>
+    <button class="modification">Modyfikuj</button>
+    <button class="deactivation">Dezaktywuj</button>
+    <button class="activation">Aktywuj</button>
+    <button class="success">Zatwierdź</button>
+    <button class="error">Odrzuć</button>
+    <button class="warning">Nie wiem</button>
+    <button class="formButton">Formularze</button>
```

Comment:
ten szary background na hover bardzo tutaj nie pasuje; przegadajmy to na kanale i zmienmy